src/main/java/org/dcsa/bkg/service/BookingService.java

```diff
@@ -23,4 +23,13 @@
 
   Mono<Page<ShipmentSummaryTO>> getShipmentSummaries(
       DocumentStatus documentStatus, Pageable pageable);
+
+  default Mono<BookingResponseTO> toBookingResponseTO(BookingTO bookingTO) {
```

Comment:
This is a good idea since it needs to be reused. Please make sure the code in BookingServiceImpl (cancelBookingByCarrierBookingReference) also uses this code

Reply:
...and ideally in the PUT (update) booking as well

Reply:
`cancelBookingByCarrierBookingReference` already returns BookingResponseTO. The issue with POST was that there were a lot of test cases that checked the returned model and changing the return type would break all of them, which wasn't the case with cancellation. But probably a good idea to do this for PUT as well.

Reply:
Done